packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -48,7 +48,7 @@ export interface AuthorEditPanelProps {
 export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
   const [name, setName] = useState('')
   const [slug, setSlug] = useState('')
-  const [jobTitle, setJobTitle] = useState<Maybe<string>>()
+  const [jobTitle, setJobTitle] = useState('')
```

Comment:
React threw an error because a form value was changed from undefined to string (which caused it from going uncontrolled to controlled), this should not happen according to the error and https://reactjs.org/docs/forms.html#controlled-components. It should start out being a string.

The same thing happens in the `userEditPanel.tsx` below